src/client/src/index.tsx

```diff
@@ -12,6 +12,11 @@ const MainRoute = lazy(() => import('./apps/MainRoute'))
 const StyleguideRoute = lazy(() => import('./apps/StyleguideRoute'))
 const SimpleLauncher = lazy(() => import('./apps/SimpleLauncher'))
 
+window.addEventListener('beforeinstallprompt', e => {
+  e.preventDefault()
+  window.beforeInstallPromptEvent = e
+})
+
```

Comment:
Is there a nicer way to do this @steinalex and @mozmorris ? It makes me a bit uncomfortable to see an `addEventListener` in a tsx file, but I know you are just capturing the event for later use...

Reply:
@steinalex did have this inside his hook, but it wasn't reliably triggered - I'd assume by the time the hook was "rendered", the event had already fired.

Reply:
I did mention to Alex we could try moving it to `MainRoute.tsx -> useInstallPromot()` but then you're back to hoping that's it's captured.

Reply:
I think in this instance (and after a lot of testing) it would be best to keep this in the `index.tsx`. Otherwise there's the risk of it not being capured